src/globus_cli/services/transfer/recursive_ls.py

```diff
@@ -60,13 +58,11 @@ def __init__(
         ls_params: dict[str, t.Any],
         *,
         max_depth: int = 3,
-        filter_after_first: bool = True,
```

Comment:
Was this param simply unused? I can't recall why we needed it, but it seems like maybe just a vestige (in which case, 👍 for removing it!).

Reply:
Yeah, I was worried it was something I'd have to figure out how to support, but it was always passed as True, so I just removed it